editor/inspector/editor_resource_tooltip_plugins.cpp

```diff
@@ -93,6 +93,23 @@ VBoxContainer *EditorResourceTooltipPlugin::make_default_tooltip(const String &p
 	return vb;
 }
 
+void EditorResourceTooltipPlugin::append_editor_description_tooltip(const String &p_resource_path, Control *p_base) {
+	if (ResourceLoader::exists(p_resource_path)) {
+		String editor_description = ResourceLoader::load(p_resource_path)->get_editor_description();
```

Comment:
If the resource is heavy (e.g. an imported scene or 4k texture), this will freeze the editor every time the tooltip is loading. The EditorFileSystem should cache whether the resource has editor description or not.